src/integrationTest/resources/tests/transcriptions/transcription/expectedResponse.json

```diff
@@ -2,6 +2,17 @@
   "case_id": 1,
   "case_number": "1",
   "courthouse": "some-courthouse",
+  "status": "APPROVED",
+  "from": "IntegrationTest User",
+  "received": "2023-06-20T10:00:00Z",
+  "comments": [
```

Comment:
TBH I was only expecting a single comment here (0:1) at transcription_status: 1 | Requested
To coincide with the screenshot @bpires2 sent on [Slack](https://hmcts-reform.slack.com/archives/C04T0CVJ28N/p1700154441901979?thread_ts=1700129032.750589&cid=C04T0CVJ28N) for the Request details > Instructions